test/functional/comment_controller_test.rb

```diff
@@ -258,4 +258,12 @@ def teardown
         type: 'page'
     assert ActionMailer::Base.deliveries.collect(&:subject).include?("New comment on 'Wiki page title'")
   end
+
+  test 'should prompt user if comment includes question mark' do
+    UserSession.create(rusers(:jeff))
+    xhr :post, :create,
+        id: comment.id,
+        body: 'Test question?'
+    assert_select 'a[href=?]', '/questions', { :count => 1, :text => 'Questions page' }
```

Comment:
Here I'm worried that you may not get this to pass, if you're using `xhr :post, :create,` above, since you'll notice we route differently based on `xhr`. But maybe you're right and the partial template that's sent back will contain the correct `a` tag:

https://github.com/publiclab/plots2/blob/master/app/controllers/comment_controller.rb#L29

Let's see what happens!